shared/local-debug.desktop.js

```diff
@@ -1,12 +1,12 @@
+// @flow
 /*
  * File to stash local debug changes to. Never check this in with changes
  */
 
-import {createRouterState} from './reducers/router'
```

Comment:
this took a little while to debug. turns out config is undefined since we have this circular dependency. instead of this pulling in a specific util from router it should just return its data simply and router should do this transform itself